requirements-test-libraries.txt

```diff
@@ -249,7 +249,8 @@ zarr==3.0.5; python_version >= '3.11'
 intake==2.0.8; python_version >= '3.9'
 h3==4.2.2
 selectolax==0.3.28
- ruamel.yaml.string==0.1.1
+ruamel.yaml.string==0.1.1
+niquests>=3,<4
```

Comment:
We tend to pin the latest version here, because we have pyup-bot weekly update these entries and run tests for them. That way, we can see what packages need updated hooks for their new releases.

Reply:
done